test/ng/compileSpec.js

```diff
@@ -12111,6 +12111,620 @@ describe('$compile', function() {
     });
   });
 
+  describe('ngOn* event binding', function() {
+    it("should add event listener of specified name", inject(function($compile, $rootScope) {
+      $rootScope.name = 'Misko';
+      element = $compile('<span ng-on-foo="name = name + 3"></span>')($rootScope);
+      element.triggerHandler('foo');
+      expect($rootScope.name).toBe('Misko3');
+    }));
+
+    it("should use angular.element(x).on() API to add listener", inject(function($compile, $rootScope) {
```

Comment:
`...to add listener` --> `...to add listener, so that Jason can hook his ngUpgrade hacks into it` 😛

Reply:
:rofl: